sql/core/src/test/scala/org/apache/spark/sql/SparkSessionJobTaggingAndCancellationSuite.scala

```diff
@@ -112,7 +112,8 @@ class SparkSessionJobTaggingAndCancellationSuite
         activeJob.properties.get(SQLExecution.EXECUTION_ROOT_ID_KEY).asInstanceOf[String].toLong)))
   }
 
-  test("Cancellation APIs in SparkSession are isolated") {
+  // TODO(SPARK-50205): Re-enable this test case.
```

Comment:
It seems that you are confused about the usage of ID-ed TODO comment, @xupefei . :)

The ID part of IDed TODO should have an open JIRA issue in order not to forget this.

For example, SPARK-48138 added a comment, `TODO(SPARK-48139)`, not `TODO(SPARK-48138)`.
```
SPARK-48138 Disable a flaky `SparkSessionE2ESuite.interrupt tag` test
SPARK-48139 Re-enable `SparkSessionE2ESuite.interrupt tag`
```

Reply:
Thanks for the explanation!